CHANGELOG/CHANGELOG-2.5.md

```diff
@@ -0,0 +1,77 @@
+- Migrate dependency update configuration from Dependabot to Renovate to unify dependency management. ([#7354](https://github.com/antrea-io/antrea/pull/7354), [@ghadeer-elsalhawy])
+- Remove `AntreaProxy` and `NodePortLocal` feature gates from Helm charts and standard manifests. ([#7505](https://github.com/antrea-io/antrea/pull/7505), [@hongliangl])
+- Add validation to ensure that `AntreaIPAM` is enabled with `SecondaryNetwork`. ([#7556](https://github.com/antrea-io/antrea/pull/7556), [@luolanzone])
+- Periodically sync ip rules managed by Antrea to ensure consistency. ([#7295](https://github.com/antrea-io/antrea/pull/7295), [@hongliangl])
+- Use a more robust way to extract the source Node IP from encapsulated IGMP messages for multicast when the traffic mode is Encap. ([#7282](https://github.com/antrea-io/antrea/pull/7282), [@hongliangl])
+- Add validation to clarify `networkPolicyOnly` mode is not supported with Antrea Multicast. ([#7362](https://github.com/antrea-io/antrea/pull/7362), [@wenyingd])
+- Unify validation logic for IPPool and ExternalIPPool for more consistent checks and failures. ([#7319](https://github.com/antrea-io/antrea/pull/7319), [@wenqiq])
+
+### Fixed
+
+- Fix overflow issue in network policy priority assigner. ([#7496](https://github.com/antrea-io/antrea/pull/7496), @Dyanngg)
+- Add missing policy UIDs for denied connections for FlowExporter. ([#7388](https://github.com/antrea-io/antrea/pull/7388), [@antoninbas])
+- Upgrade `libOpenflow` to support OpenFlow message overflow. ([#7470](https://github.com/antrea-io/antrea/pull/7470), [@wenyingd])
+- Fix ServiceCIDR discovery in the Multi-cluster member controller for Kubernetes versions 1.33 and newer. ([#7291](https://github.com/antrea-io/antrea/pull/7291), [@luolanzone])
+- Fix Agent crash issue which is caused by unexpected interface store initialization for FlexibleIPAM uplink internal port. ([#7389](https://github.com/antrea-io/antrea/pull/7389), [@gran-vmv])
+- Avoid missing or invalid NetworkPolicy data in FlowExporter records by increasing flow ID reuse delay and filtering old connections. ([#7468](https://github.com/antrea-io/antrea/pull/7468), [@antoninbas])
+- Refine Traceflow to correctly handle inter-Node Pod-to-Pod traffic across all traffic modes. ([#7481](https://github.com/antrea-io/antrea/pull/7481), [@hongliangl])
+- Fix ACNP applied to NodePort failing to reject traffic when the traffic mode is `noEncap` or `hybrid`. ([#7265](https://github.com/antrea-io/antrea/pull/7265), [@hongliangl])
+- Fix a type assertion panic in `GetFlowTableID` function, which affected the `/ovsflows` HTTP handler. ([#7515](https://github.com/antrea-io/antrea/pull/7515), [@antoninbas])
+- Handle missing Pod IP and Pod IP changes in NodePortLocal to prevent incorrect datapath rules and clear NPL annotations. ([#7512](https://github.com/antrea-io/antrea/pull/7512), [@antoninbas])
```

Comment:
remove "and clear NPL annotations", I think it can create confusion taken out of context